### PR TITLE
march_launch description

### LINKED ISSUE #37: march_launch
- [x] Explain that this contains the main launch file
- [x] Software checks, how they work, how to extend them

### PR DESCRIPTION
**Pull request by Ishadijcks**
Closes #37

### CODE REVIEW

File: doc/march_packages/march_launch.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+^^^^^^^^^^
+If a specific script is written to calculate the result of a check,
+it is convention to upload it to ``/check/<check_name>`` (e.g. ``/check/slave_count``)
+
+Tutorials
+---------
+
+Adding a software check
+^^^^^^^^^^^^^^^^^^^^^^^
+To add a software check, it should b
```
Comment (JitskedeVries) [inline doc/march_packages/march_launch.rst]: Unfinished sentence

**Review by JitskedeVries — CHANGES_REQUESTED** [review]
Looks good, only one unfinished sentence

File: doc/march_packages/march_launch.rst (hunk 1 above)
Comment (Ishadijcks) [reply doc/march_packages/march_launch.rst]: Woops

Comment (Ishadijcks) [inline doc/march_packages/march_launch.rst]:
```suggestion
When adding a software check for the first time, make sure to take a look at existing software checks. The ``SlaveCountCheck`` especially is suitable as it runs on another machine. The following steps will guide you through the process.
```

**Review by JitskedeVries — APPROVED** [review]

**Pull request merged**